frontend/src/pages/MapView/styles.scss

```diff
@@ -7,17 +7,18 @@
     left: 12px;
     top: 72px;
     width: 342px;
-    height: 70%;
+    height: auto;
     z-index: 1;
 }
 
 .cardContent {

```

Comment:
Nit: I know this was an artifact of a previous PR, but can we change this to having words separated with dashes like you did above (which is correct according to CSS style spec)? So `cardContent` -> `card-content`.